myhpi/settings.py

```diff
@@ -84,6 +85,7 @@
 LOGIN_URL = "login"
 
 MIDDLEWARE = [
+    # "debug_toolbar.middleware.DebugToolbarMiddleware", # Enable this line to get debug toolbar
```

Comment:
Perhaps add an env variable to automatically add this to middleware?

Reply:
done